### ISSUE TITLE
"Very high storage usage after downgrading." and "Jailbreak's don't work after downgrading." can be fixed.

### ISSUE THREAD

**Issue by Halo-Michael** [MatthewPierson/PyBoot#2]
As the title said, "Very high storage usage after downgrading." and "Jailbreak's don't work after downgrading." can be fixed. It's because of the old system snapshot is still in rootfs, and keeping take up space. You can use command "snappy -f / -l" to check that. For those people who never jailbreaded in the old system, the snapshot should named "com.apple.os.update-<hash>". And for those people who used checkra1n loader app or unc0ver, It should named "orig-fs". Checkra1n loader will check the snapshot whether it matches the current system version. So we just need to delete this snapshot to fix the unjailbreakable issue and release the space.
To delete the snapshot (whatever the name is), you should run command "snappy -f / -d $(snappy -f / -l | sed -n '2p')".

**Comment by narcyzzo** [comment]
how to do this ? i must install snappy tweak but how when i cant install cydia? or i can manually delete snapshot with root acces ?

**Comment by Xoloth** [comment]
I think you need to do that before you downgrade.

**Comment by Halo-Michael** [comment]
> how to do this ? i must install snappy tweak but how when i cant install cydia? or i can manually delete snapshot with root acces ?

checkra1n is ready for everything. Just connect the device via ssh on port 44 and run the command.

**Comment by narcyzzo** [comment]
yes, i can confirm this method with deleting snapshot working, also u can jailbreak with checkra1n version 0.9.2, this version will allow installing cydia, then u can boot with 0.9.8 and all working fine

**Comment by Hzlyy** [comment]
-bash-3.2# snappy -f / -d $(snappy -f / -l | sed -n '2p')

Will delete snapshot named orig-fs from fs /
fs_snapshot_delete: Read-only file system
Failure

**Comment by Hzlyy** [comment]
I did, but when I tried to install cydia, I got "bootstrap error: unable to rename snapshot"
How can I solve this problem?

**Comment by Hzlyy** [comment]
> yes, i can confirm this method with deleting snapshot working, also u can jailbreak with checkra1n version 0.9.2, this version will allow installing cydia, then u can boot with 0.9.8 and all working fine

But I tried to install the plug-in after I escaped from prison. When I installed the substrate safe mode and Cydia substrate, the card owner will be logged out and restart automatically. Then when I try to checkra1n boot again, it will show xxxxxxxxxxx(substrated). How can I solve this problem? I have rebooted the machine more than 20 times....
Please help me. I've been working on it for several days...
